packages/dev/core/src/Rendering/renderingGroup.ts

```diff
@@ -242,7 +242,9 @@ export class RenderingGroup {
             for (; subIndex < subMeshes.length; subIndex++) {
                 subMesh = subMeshes.data[subIndex];
                 subMesh._alphaIndex = subMesh.getMesh().alphaIndex;
-                subMesh._distanceToCamera = Vector3.Distance(subMesh.getBoundingInfo().boundingSphere.centerWorld, cameraPosition);
+                if(subMesh.getBoundingInfo()){
+                    subMesh._distanceToCamera = Vector3.Distance(subMesh.getBoundingInfo().boundingSphere.centerWorld, cameraPosition);

```

Comment:
```suggestion
                    const boundingInfo = subMesh.getBoundingInfo();
                    subMesh._distanceToCamera = boundingInfo ? Vector3.Distance(boundingInfo.boundingSphere.centerWorld, cameraPosition) : 0;

```
We should set a value to `_distanceToCamera` in all cases, else the sorting may fail later on.

Reply:
Yes! This is what we need.